cmd/nvidia-device-plugin/main.go

```diff
@@ -196,8 +203,10 @@ func loadConfig(c *cli.Context, flags []cli.Flag) (*spec.Config, error) {
 }
 
 func start(c *cli.Context, flags []cli.Flag) error {
-	klog.Info("Starting FS watcher.")
-	watcher, err := watch.Files(pluginapi.DevicePluginPath)
+	kubeletSocket := c.String("kubelet-socket")
+	kubeletSocketDir := filepath.Dir(kubeletSocket)
+	klog.Infof("Starting FS watcher for %v", kubeletSocketDir)
```

Comment:
```suggestion
	klog.Infof("Starting FS watcher for %s", kubeletSocketDir)
```

Reply:
Since `kubeletSocketDir` is a `string` these two should be equivalent. Is there a specific reason to prefer `%s` over `%v` in this case?

Reply:
This is just a minor nit. I prefer `%s` as it explicitly says that variable is a string